### PR TITLE
Updated to node20 and dependencies

### LINKED ISSUE #22: Updated node20 and dependencies

### PR DESCRIPTION
**Pull request by jpolavar**
Closes #22

### CODE REVIEW

File: .github/workflows/publish.yml
Comment (adcreare) [inline .github/workflows/publish.yml:1]: This needs to be updated to the "standard" publish action for a public repo

**Review by adcreare — CHANGES_REQUESTED** [review]
Looks good - publish action needs to be updated to the latest action for a public repo.

**Review by adcreare — APPROVED** [review]
LGTM

**Review by adcreare — APPROVED** [review]
LGTM

**Comment by jpolavar** [comment]
@adcreare https://github.com/checkdigit/hash/actions/runs/8442321597/job/23123476131 seems audit signature failed

**Pull request merged**